apps/sv/frontend/src/components/forms/GrantRevokeFeaturedAppForm.tsx

```diff
@@ -50,17 +51,25 @@ export const GrantRevokeFeaturedAppForm: React.FC<GrantRevokeFeaturedAppFormProp
   const dsoInfosQuery = useDsoInfos();
   const initialExpiration = getInitialExpiration(dsoInfosQuery.data);
   const initialEffectiveDate = dayjs(initialExpiration).add(1, 'day');
+  const [showConfirmation, setShowConfirmation] = useState(false);
```

Comment:
Every form has to hold this state. I could lift it to a higher component but I don't think the juice is worth the squeeze.